index.html

```diff
@@ -24,10 +24,73 @@ <h1 id="title">5e Spellbook</h1>
       </div>
     </header>
     <main>
-      <div class="view" data-view="all-spells-list">
+      <!-- SPELLS LIST VIEW -->
+      <div id="spells-list-view" class="view">
         <h1>Spells List</h1>
         <div id="spells-list-cards-div" class="row"></div>
       </div>
+
+      <!-- SPELL DETAILS VIEW -->
+      <div id="spell-details-view" class="view hidden">
+        <a id="spell-details-back-anchor" class="hover-pointer">
+          <i class="fa-solid fa-chevron-left"></i>
+          Back
+        </a>
+        <div id="spell-scroll-page">
+          <div id="spell-scroll-top">
+            <div class="scroll-end scroll-end-dark"></div>
+            <div class="scroll-end scroll-end-light"></div>
+          </div>
+          <div id="all-spell-details">
+            <div class="spell-details-group">
+              <h1 id="spell-details-name">Spell Name</h1>
+              <div>
+                <span id="spell-details-level-school">level and school</span>
+              </div>
+            </div>
+            <div class="spell-details-group">
+              <div>
+                <span style="font-weight: 700">Casting Time: </span>
+                <span id="spell-details-cast-time">cast time</span>
+              </div>
+              <div>
+                <span style="font-weight: 700">Range: </span>
+                <span id="spell-details-range">range</span>
+              </div>
+              <div>
+                <span style="font-weight: 700">Components: </span>
+                <span id="spell-details-components">components</span>
+              </div>
+              <div>
+                <span style="font-weight: 700">Duration: </span>
+                <span id="spell-details-duration">duration</span>
+              </div>
```

Comment:
On all of the second `span` elements, do you need base text content on them? Is this being replaced by your TS?

Reply:
Yes, the text is being replaced. I mostly only put that there so I could see the changes as I wrote my TS but I will remove them.